modules/react/src/utils/evaluate-children.ts

```diff
@@ -4,7 +4,7 @@ import {cloneElement} from 'react';
 const MAP_STYLE = {position: 'absolute', zIndex: -1};
 
 export default function evaluateChildren(
-  children: React.ReactNode,
+  children: React.ReactNode | Function,
```

Comment:
Just nits/questions:
- This is related to the yarn.lock bump to React 18 types? 
- Is that related to this PR or just an accident that could have been a separate PR?
- Is it possible to define a more specific function type instead of  just `Function`?
```
type EvaluateChildren = (children: ...) => ...
```

Reply:
The above change is required to run `tsc` successfully on CI, but I don't see the error locally.
Regarding more accurate typing for `children` - I will open a separate PR. The current type does not support callbacks, JSX layers or JSX views. To fix this will be more than a few lines of change. There are a couple of related open issues.